.github/workflows-src/jobs.lib.yml

```diff
@@ -47,6 +47,7 @@ steps:
   - #@ steps_genupdate()
   - #@ steps_build_state_msi()
   - #@ steps_build_languages_msi()
+  - #@ steps_msi_inttest()
```

Comment:
The MSI language integration tests depend on the MSI, of course.  So I created a separate step for them and put them into the "build" job on Windows.  It's not very pretty, as a "test" is not a "build", which is why I filed a follow-up story: https://www.pivotaltracker.com/story/show/174260391